base/replutil.jl

```diff
@@ -193,10 +193,26 @@ function showerror(io::IO, ex::TypeError)
 end
 
 function showerror(io::IO, ex, bt; backtrace=true)
-    try
-        showerror(io, ex)
+   try
+        Base.with_output_color(:red, io) do io
+            showerror(io, ex)
+        end
     finally
-        backtrace && show_backtrace(io, bt)
+        if backtrace
+            if get(io, :REPLError, false)
```

Comment:
Similarly, since the call to `show_backtrace` is directly from here, I think you can just pass format as a kwarg there.